src/vs/workbench/contrib/debug/browser/debugANSIHandling.ts

```diff
@@ -15,6 +15,8 @@ export function handleANSIOutput(text: string, linkDetector: LinkDetector): HTML
 	const textLength: number = text.length;
 
 	let styleNames: string[] = [];
+	let customFgColor: RGBColor | null = null;
```

Comment:
We prefer using `undefined` to `null`. So please try using `undefined` instead

Reply:
Fixed with 5096a35.